integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/detailquery/SearchModeTestCase.scala

```diff
@@ -111,22 +111,26 @@ class SearchModeTestCase extends QueryTest with BeforeAndAfterAll {
   }
 
   test("test lucene datamap with search mode") {
+    sql("set carbon.search.enabled = true")
```

Comment:
Is this required? Search mode is already started

Reply:
Required. Because the previous test case close the search mode by set